packages/manager/src/utilities/omittedProps/omittedProps.ts

```diff
@@ -3,16 +3,13 @@
  * passing them down to the DOM. This is to avoid the verbose API presented by
  * MUI and Emotion and to provide type safety.
  * @param props Array of props to filter out
- * @param prop The prop to check
- * @returns Boolean indicating whether the prop is valid
+ * @returns Boolean indicating whether the prop should be omitted
  * @see https://mui.com/material-ui/customization/how-to-customize/#dynamic-overrides
  * @Usage
  *    const MyComponent = styled(Button, {
- *      shouldForwardProp: (prop) =>
- *      isPropValid(['compactX', 'compactY'], prop),
+ *      shouldForwardProp: omittedProps(['compactX', 'compactY']),
  *    })<Props>(({ theme, ...props }) => ({ ... }));
  */
-export const isPropValid = <CustomProps extends Record<string, unknown>>(
-  props: Array<keyof CustomProps>,
-  prop: PropertyKey
-): boolean => !props.includes(prop as string);
+export const omittedProps = <CustomProps extends Record<string, unknown>>(
+  props: Array<keyof CustomProps>
+) => (prop: PropertyKey): boolean => !props.includes(prop as string);
```

Comment:
Although we'd have a specify a generic type to reap the benefits, could this change help us catch errors by improving the type-safety?

```suggestion
export const omittedProps = <Props>(props: Array<keyof Props>) => (
  prop: keyof Props
): boolean => !props.includes(prop);
```

![Screenshot 2023-10-12 at 3 47 08 PM](https://github.com/linode/manager/assets/115251059/7a5cca3b-101f-4064-93f9-960d4f4e68c4)
![Screenshot 2023-10-12 at 3 48 38 PM](https://github.com/linode/manager/assets/115251059/203a71a0-b58c-4745-a071-631cf6e22199)

Reply:
sure thing - it won't hurt for typos 👍

Reply:
done!

Reply:
updated the unit test to be a real case too